charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -20,3 +20,1475 @@ spec:
+                          type: integer
+                        successThreshold:
+                          type: integer
+                        timeoutSeconds:
+                          type: integer
+                    nodeSelector:
+                      type: object
+                    rbac:
+                      type: object
+                      required:
+                      - createClusterMonitoringViewRBAC
+                      properties:
+                        createClusterMonitoringViewRBAC:
+                          type: boolean
+                    readinessProbe:
+                      type: object
+                      properties:
+                        failureThreshold:
+                          type: integer
+                        httpGet:
+                          type: object
+                          required:
+                          - path
+                          - port
+                          - scheme
+                          properties:
+                            path:
+                              type: string
+                            port:
+                              type: integer
+                            scheme:
+                              type: string
+                        initialDelaySeconds:
+                          type: integer
+                        periodSeconds:
+                          type: integer
+                        successThreshold:
+                          type: integer
+                        timeoutSeconds:
+                          type: integer
+                    replicas:
+                      type: string
+                    resources:
+                      type: object
+                      properties:
+                        limits:
+                          type: object
+                          properties:
+                            cpu:
+                              type: string
+                            memory:
+                              type: string
+                        requests:
+                          type: object
+                          properties:
+                            cpu:
+                              type: string
+                            memory:
+                              type: string
+                    route:
+                      type: object
+                      required:
+                      - enabled
+                      properties:
+                        enabled:
+                          type: boolean
+                        name:
+                          type: string
+                    tolerations:
+                      type: array
+                    updateStrategy:
+                      type: object
+                      required:
+                      - type
+                      properties:
+                        type:
+                          type: string
+            storage:
+              type: object
+              properties:
```

Comment:
This is gonna need the other properties otherwise when we merge this it'll fail right? Eg: s3, azure, sharedPVC